### ISSUE TITLE
[Support]: Video starting after event is over

### ISSUE THREAD

**Issue by kdfha** [blakeblackshear/frigate#2213]
### Describe the problem you are having

Video often not catching the event. Video starts after event as shown by camera timestamp in the snapshot. Changing pre_capture doesn't seem to affect that. 

Snap shown on events, camera time stamp 2021-11-09 14:14:21 :

![Capture](https://user-images.githubusercontent.com/92119205/141029052-004a9829-bb41-4f16-afa4-de54644dcb77.JPG)

Snip of first frame of video, camera time stamp 2021-11-09 14:14:46

![Capture2](https://user-images.githubusercontent.com/92119205/141029370-243a3fd7-1e70-4c4a-afe1-bbce3c82cefe.JPG)

The event video started 25 seconds after the time in the snapshot event. In spite of it being set to pre_capture 10 seconds before. 

### Version

DEBUG 0.9.0-8CA1280

### Frigate config file

```yaml
mqtt:
  host: XXX.XXX.XXX.XXX
  
detectors:
   coral:
    type: edgetpu
    device: usb

cameras:
  driveway:
    ffmpeg:
      inputs:
        - path: rtsp://XXX.XXX.XXX.XXX:554/user=XXXX=XXXXXX_channel=1_stream=0.sdp?real_stream
          roles:
            - detect
            - record
    detect:
      fps: 7
    motion:
      mask:
        - 889,117,1208,119,1208,87,893,87
    snapshots:
      enabled: True
      bounding_box: True
      retain:
        default: 30


  shed:
    ffmpeg:
      inputs:
        - path: rtsp://XXX.XXX.XXX.XXX/1/h264major
          roles:
            - detect
            - record
    detect:
      fps: 7
    motion:
      mask:
        - 307,929,0,936,0,899,311,895
        - 1092,408,1174,411,1205,0,1075,0
    snapshots:
      enabled: True
      bounding_box: True
record:
  enabled: True
  retain_days: 0
  events:
    max_seconds: 300
    pre_capture: 10
    post_capture: 10
    retain:
      default: 30
     
objects:
  track:
    - person
    - bird
    - cat
    - dog
    - bicycle
    - deer
    - motorcycle
birdseye:
  mode: continuous
```

### Relevant log output

```shell
Um, have restarted since last problem. Nothing in log.
```

### FFprobe output from your camera

```shell
Not sure how to do that from Home Assistant.
```

### Frigate stats

```json
Not sure where to find that in Home Assistant.
```

### Operating system

HassOS

### Install method

HassOS Addon

### Coral version

USB

### Network connection

Wired

### Camera make and model

Happens on both cameras. Different brands.

**Comment by blakeblackshear** [comment]
Is there a specific reason you haven't updated to 0.9.4? There have been some related issues addressed since 0.9.0.

**Comment by kdfha** [comment]
I get what shows up in Home Assistant using the repository listed in the install instructions. 

https://github.com/blakeblackshear/frigate-hass-addons

The OS and Core lets me know when there is an update available. I click Update and away it goes. 

There hasn't been the option to update Frigate. How does one update?

Thanks
Ken
